### PR TITLE
Accept message as value and return key (or error)

### PR DESCRIPTION
**Pull request by mycognosist**
- [x] Export message-related structs from ssb-validate (https://github.com/sunrise-choir/ssb-validate/pull/5)
  - Turns out I didn't actually need all of this but it'll probably be useful down the line
- [x] Convert `validateSingle` to take a message value instead of a KVT message
- [x] Use `ssb_validate::multihash_from_bytes(&msg_bytes)` to generate `key` from `value`
- [x] Return tuple from each Rust validation function (with `key` as `Option<Vec<String>>` and `err_msg` as `Option<String>`)
- [x] Update `index.js` to destructure returned list (tuple in Rust) to `[err, res]`, where `res` is a list holding the `key`(s)
- [x] Add OOO and multi-author support for message `value` to `ssb-validate`
- [x] Refactor multi-author and out-of-order functions to take message values
- [x] Update test(s)
- [x] Ensure CI tests pass
- [x] Do a 'lil dance

Related to https://github.com/ssb-ngi-pointer/ssb-validate2-rsjs/issues/26

Made possible by https://github.com/sunrise-choir/ssb-validate/pull/9

### CODE REVIEW

**Review by staltz — APPROVED** [review]
YUP :heavy_check_mark:

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ssb-validate2-rsjs-node",
-  "version": "0.5.1",
+  "version": "0.6.0",
```
Comment (staltz) [inline package.json]: @mycognosist Actually, I just realized, I think this should not be bumped yet. It should be bumped only its own commit message that starts with `"release"`, e.g. `git commit -m'release 0.6.0'` which will trigger CI to do the version release on npm for us.

Comment (mycognosist) [reply package.json]: Ah yes, good call. I made a mess of the revert but it's resolved now.

**Comment by arj03** [comment]
Wooo, even `Do a 'lil dance` ticket off :-) We need videos for next weekly meeting ;-)

**Pull request merged**